src/Layout.tsx

```diff
@@ -105,7 +105,7 @@ const Layout: React.FC<LayoutProps> = (props) => {
           content={`${siteUrl}${image ?? defaultImage}`}
         />
         <meta
-          property="description"
+          name="description"
```

Comment:
We still have other `<meta property=...` tags above this one; per MDN, `property` isn't a supported attribute on `<meta>` tags: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta

Reply:
it's what Open Graph Protocol uses so go yell at them

if you remove them embeds will break
https://ogp.me/

![image](https://user-images.githubusercontent.com/46435130/139602842-c8a15601-a58b-4153-99c6-d6e3bc8b8b68.png)

Reply:
As the kids say, that is *wack*

Reply:
Straight wack, yo no cap